package.json

```diff
@@ -496,7 +493,7 @@
   "scripts": {
     "lint": "eslint . --ext js --ignore-pattern *.tpl.js",
     "lint:fix": "eslint . --ext js --ignore-pattern *.tpl.js --fix",
-    "test:unit": "grunt",
+    "test:unit": "mocha 'test/unit/**/*.spec.js'",
```

Comment:
the single quotes should be double quotes (escaped) otherwise this won't work in all shells.

Reply:
Good point, thanks!